mypy.ini

```diff
@@ -4,6 +4,9 @@ follow_imports = normal
 strict = False
 no_incremental = True
 
+[mypy-Cython.*]
```

Comment:
Can you add a comment to the top of this config file saying what we use it for, and what mypy is?

Reply:
Thanks!
I added the docs to make it clear.